packages/blade/public/universal/types.ts

```diff
@@ -31,5 +31,6 @@ export type {
 
 export type * from 'blade-client';
 export type * from 'blade-client/types';
-export type { ResultRecord } from 'blade-compiler';
-export type { ReducedFunction } from 'blade-syntax/queries';
+
+export type ReducedFunction = any;
+export type ResultRecord = any;
```

Comment:
Can we not at least copy / inline the types ourselves & mark a TODO to re-add the re-export later on, since for whatever reason `tsdown` is not able to resolve them.

That way we don't regress all the work just merged recently to make types better.